keyboards/orange75/info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "Orange75", 
+    "url": "", 
+    "maintainer": "qmk", 
+    "bootloader": "", 
```

Comment:
This line can be deleted as the object's usage here is deprecated.

Reply:
which line?